### ISSUE TITLE
Warnings: Add Warning when Vault and Strategy Versions are not compatible

### ISSUE THREAD

**Issue by salazarguille** [yearn/yearn-watch-legacy#104]
Add a warning when the vault version and strategy versions are not compatible in Vaults home and vault Page.

**Comment by uwe** [comment]
What are the rules for "not compatible"? Different minor version, eg. 0.4.0 vs. 0.3.0?

**Comment by storming0x** [comment]
> What are the rules for "not compatible"? Different minor version, eg. 0.4.0 vs. 0.3.0?

Yes, different minor versions, so 0.4.3 w 0.4 works, and 0.3 vault with 0.3 are good.

**Comment by saltyfacu** [comment]
Will add this to the alerts section of the new UI

**Comment by Majorfi** [comment]
Will be handled by #270
